corehq/apps/app_manager/suite_xml.py

```diff
@@ -748,20 +761,30 @@ def add_form_workflow(self, suite):
           * Finally remove the last item from the stack frame.
         """
         from corehq.apps.app_manager.models import (
-            WORKFLOW_DEFAULT, WORKFLOW_PREVIOUS, WORKFLOW_MODULE, WORKFLOW_ROOT
+            WORKFLOW_DEFAULT, WORKFLOW_PREVIOUS, WORKFLOW_MODULE, WORKFLOW_ROOT, WORKFLOW_FORM
```

Comment:
Would like to refactor this function to get rid of all the internal functions but will leave it till after this is merged otherwise it'll be hard to review.